pythonparser/ast.py

```diff
@@ -57,6 +57,15 @@ def value(name):
                            self._reprfields()))
         return "%s(%s)" % (self.__class__.__name__, fields)
 
+    # Compat with stdlib
+    @property
+    def lineno(self):
+        return self.loc
```

Comment:
I do remember ``test_ast.py`` to fail more if I remove the setter, but will try again. And I am on envy of it (``test_ast.py``) to be already testing this stuff. The test came on #16, but can be chopped to test each feature individually. I vote against for simplicity. 

You are probable right on ``self.loc.line()``. Thanks for pointing to the right direction.